Dockerfile

```diff
@@ -1,13 +0,0 @@
-# Build in a different image to keep the target image clean
-FROM node:20-alpine as build
-WORKDIR /usr/src/app
-COPY ./package.json ./package-lock.json ./
-RUN npm install
-COPY ./ ./
-RUN npm run build
-
-# The target image that will be run
-FROM nginx:alpine as target
-WORKDIR /usr/src/app
-COPY ./docker/nginx.conf /etc/nginx/nginx.conf
-COPY --from=build --chown=nginx /usr/src/app/dist/ /usr/src/app
```

Comment:
Currently, if it were used for a GitHub action, it would fail as the client generated files would not be included.

Since the client has shrunk significantly in size, I am considering adding client files to these repositories as well though. However, I want to leave that for a separate PR.

Reply:
But how can we now build a Docker image for the backoffice without a Docker file? Right now, I am building the Aurora images manually so these files are present, but it would indeed be nice to use GH Actions at some point.